OWML.Launcher/SocketListener.cs

```diff
@@ -11,7 +11,8 @@ namespace OWML.Launcher
 {
     public class SocketListener
     {
-        private const int BufferSize = 1024;
+        private const string Separator = "\n--------------------------------";
+        private const int BufferSize = 1<<18;
```

Comment:
Either add a code comment here or write this out plainly - I can see someone (or us) having to look up what this is equal to